### DISCUSSION TITLE
Randomize seed on batch generation

### DISCUSSION THREAD

**Discussion by barepixels** [MoonRide303/Fooocus-MRE#48] [Ideas]
Am reposting this from  https://github.com/lllyasviel/Fooocus/issues/306

```
The seed does not randomize between images when batch generating images, it increments by 1.

Is this by design or a bug, now it only randomizes if I generate 1 image at a time?

I can see the value with incrementing but it would be very nice to have the option to randomize between images too.
```
I just verified with batch, it increments by 1
@MoonRide303 Maybe we can have an "increment" next to the "random" option for Single and Batch

**Comment by MoonRide303** [comment]
@barepixels I've explained it in the original issue - it's only necessary for first number to be random (to avoid using popular numbers), and following seeds can be simply increased by 1. They don't need to be far away to generate completely different image.
